### DISCUSSION TITLE
授权功能

### DISCUSSION THREAD

**Discussion by yangyingchun1994** [milvus-io/milvus#31164] [Announcement]
怎么设置分区级别的权限，例如把某个collection的某个分区授权给一个用户，支持这个功能吗？

**Comment by yhmo** [comment]
这个文档里 https://milvus.io/docs/users_and_roles.md 提到的目前只有3种分类: Collection, Global, User
另外还有通过设置db_name来控制database级别的权限: https://milvus.io/docs/manage_databases.md

所以，目前还无法控制partition级别的授权。
